xml/art_sle_ha_pmremote.xml

```diff
@@ -360,7 +360,7 @@
   <sect2 xml:id="sec.ha.pmremote.config.uc1.remote.nodes">
    <title>Configuring Remote Nodes</title>
    <para>
-    The following procedure creates a guest node <systemitem
+    The following procedure prepares the remote node <systemitem
       class="domainname">&node4;</systemitem> on your cluster node
     <systemitem class="domainname">&node3;</systemitem>:
```

Comment:
Right, I've removed it, so the sentence is: "The following procedure prepares your cluster node
     <systemitem class="domainname">&node3;</systemitem>:"